README.md

```diff
@@ -100,16 +103,37 @@ cd /srv/atd-airflow;
 # pull the changes
 git pull;
 
-# return to user-land
-exit;
+
 ```
 
+#### If there has been a change to the Airflow Docker image like when updating the Airflow verson:
+
 The production Airflow deployment uses a second Docker compose file which provides haproxy configuration overrides. To start the production docker compose stack use you must load both files in order:
 
 ```shell
+# dts-int-data-p01
+
+# become the superuser
+su -;
+
+# enter into the production airflow directory
+cd /srv/atd-airflow;
+
+# pull the fresh production image
+docker pull atddocker/atd-airflow:production
+
+# stop the Docker stack
+docker compose stop
+
+# start the Docker stack
 docker compose -f docker-compose.yaml -f docker-compose-production.yaml up -d
+
```

Comment:
Maybe add a line here with the `docker compose logs -f` to tail the logs? `ctrl-c` breaks you out of that log tailing.

Reply:
yeah! that was super helpful to watch as the stack was coming up. i'll add.